ardriveapp/ardrive-web
# PE-1078: Move string literals into resources file

This PR moves all string literals into the ARB file.

**How to review**
- At the codebase:
  - Ensure the text is replaced with the correct internationalization variable
  - Ensure the Text that was in upper case calls `toUpperCase()` after the replacement
- At the ARB file:
  - Check that the description gives the right context information to understand the text
    - Check for ellipsis: is the substantive omitted? If so, is it implicit in the description?
    - Is it talking about plural or singular verbs?
